geometric/params.py

```diff
@@ -48,6 +48,27 @@ class OptParams(object):
     def __init__(self, **kwargs):
         # Whether we are optimizing for a transition state. This changes a number of default parameters.
         self.transition = kwargs.get('transition', False)
+        # NEB method parameters
```

Comment:
We should think about whether NEB method parameters should belong in a separate object from geometry optimization.  I'm not saying it must be one way or the other, but there are some issues with it being here: 

- Some of these options have overlapping meanings but different default values (e.g. `maxg` vs. `converge_gmax`, `maxcyc` vs. `maxiter`).
- The choice of coordinate system should probably not be part of `OptParams` because this class should contain parameters that are independent of the coordinate system.
- `tcguess` shouldn't be here because engine-related options aren't here.

Reply:
I separated `OptParams` and `NEBParams`.